gradle/publish.gradle

```diff
@@ -69,7 +69,8 @@ ext {
     }
 }
 
-def publishedProjects = subprojects.findAll {!['util', 'driver-benchmarks', 'driver-workload-executor'].contains(it.name) }
+def nonPublishProjects = project(":util").allprojects + project(":driver-benchmarks") + project("driver-workload-executor")
```

Comment:
nonPublishProjects => nonPublishedProjects

Reply:
Changed to `projectsNotPublishedToMaven`